src/styles/variables.scss

```
@@ -0,0 +1,12 @@
+$fontFamily: "Arial", "Helvetica", sanf-serif;
+$sizeOfCeil: 100px;
+$gapOfCeil: 1px;
```

why ceil? maybe cell?
```suggestion
$cellSize: 100px;
$cellGap: 1px;
```